<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SvodNewReportDialog</class>
 <widget class="QDialog" name="SvodNewReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый отчет</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDateEdit" name="edtDate"/>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="CTableView" name="tblFormList"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btnUpdateFormList">
     <property name="text">
      <string>Обновить список форм</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
